src/types/plugin/ProtocolAPI.ts

```diff
@@ -0,0 +1,12 @@
+export interface IAsyncIteratorCreator {
+  [Symbol.asyncIterator]: () => AsyncIterator<any>;
+}
+
+export interface IFractalProtocolAPI {
+  raw(): IAsyncIteratorCreator;
+}
+
+export interface IFact {
+  url: string;
+  timestampMs: number;
+}
```

Comment:
Does this file need to exist? Seems like this should be derivable from the other file being TS.

Reply:
Only the `IFractalProtocol` interface is not needed, but I use it for the Window type. This was a pattern used in the project, I am not a huge fan, but decided to keep to it for consistency.